ModuleSystem/module_scripts_common_warp.py

```diff
@@ -0,0 +1,764 @@
+from header_common import *
+from header_operations import *
+from module_constants import *
+from header_parties import *
+from header_skills import *
+from header_mission_templates import *
+from header_items import *
+from header_triggers import *
+from header_terrain_types import *
+from header_music import *
+from ID_animations import *
+from ID_troops import *
+from ID_factions import *
+#from module_troops import *
+
+from module_info import wb_compile_switch as is_a_wb_script
+
+# Credits to Autolykos, thread https://forums.taleworlds.com/index.php?topic=335511.0
```

Comment:
What's fun is that Khamukkamu can attest that I came by myself, while learning the Module System, to the potential use of parties slots as arrays and passing scripts as parameters. Then I (or Kham, I'm not sure right now) found Autolykos work and I decided to use it instead of coding it all again myself, as there is no need to reinvent the wheel, and as TLD is open source and that has been done some time, that would help other people to understand the array code right away. And he did a good job on the shorting scripts (that I didn't had in mind implementing right away).
For what I have seen of the code, I believe that some troops were used as arrays as well, and so do the merchants' stuff work in WB, right?